### PR TITLE
Update README and docs post-23.04 release

### PR DESCRIPTION
**Pull request by mc-nv**

### CODE REVIEW

File: deploy/gke-marketplace-app/server-deployer/build_and_push.sh [hunk 1]
```diff
@@ -26,9 +26,9 @@
 
 export REGISTRY=gcr.io/$(gcloud config get-value project | tr ':' '/')
 export APP_NAME=tritonserver
-export MAJOR_VERSION=2.32
-export MINOR_VERSION=2.32.0
-export NGC_VERSION=23.03-py3
+export MAJOR_VERSION=2.31
+export MINOR_VERSION=2.31.0
+export NGC_VERSION=23.02-py3
```
Comment (Tabrizian) [inline deploy/gke-marketplace-app/server-deployer/build_and_push.sh:31]: 23.04

File: deploy/gke-marketplace-app/server-deployer/chart/triton/Chart.yaml [hunk 2]
```diff
@@ -25,7 +25,7 @@
 # OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
 apiVersion: v1
-appVersion: "2.32"
+appVersion: "2.31"
```
Comment (Tabrizian) [inline deploy/gke-marketplace-app/server-deployer/chart/triton/Chart.yaml:28]: 2.33

**Review by Tabrizian — CHANGES_REQUESTED** [review]
Looks like there are a lot of errors in the version numbers.

**Review by Tabrizian — APPROVED** [review]

**Review by nnshah1 — APPROVED** [review]
All changes in docs included here - will close other MR

**Pull request merged**